atc/api/accessor/accessor.go

```diff
@@ -18,16 +18,7 @@ type Access interface {
 	IsSystem() bool
 	TeamNames() []string
 	TeamRoles() map[string][]string
-	Claims() Claims
-}
-
-type Claims struct {
-	Sub               string
-	UserID            string
-	UserName          string
-	PreferredUsername string
-	Email             string
-	Connector         string
+	Claims() atc.Claims
```

Comment:
Changed to use `atc.UserInfo`.

I never used `fly userinfo` before, but when I tried this command today, I noticed that, when I used an local account, "username" column showed nothing, which is because "local" connector doesn't populate `preferredUsername`. I have fixed this bug by adding `UserInfo.PreferredUserId()` function.